.prettierignore

```diff
@@ -7,3 +7,5 @@ templates
 plugins/scaffolder-backend/sample-templates
 .vscode
 dist-types
+packages/app/public/oauth2-redirect.html
+packages/create-app/templates/default-app/packages/app/public/oauth2-redirect.html
```

Comment:
Given this add, it made me realize, there's no `.prettierignore` in a newly scaffolded app driven from `npx @backstage/create-app`.

Do you know does a freshly templated app using your new code immediately fail a prettier check? if so we may want to add prettier ignore to the template - something i hadn't considered before.

If I run it on an app I recently scaffolded, I see:

```
~/github/my-backstage/app(add-k8s)*$ yarn prettier --check *
yarn run v1.22.5
$ /home/user/github/my-backstage/app/node_modules/.bin/prettier --check README.md app-config.local.yaml app-config.production.yaml app-config.yaml catalog-info.yaml dist-types lerna.json node_modules package.json packages tsconfig.json yarn.lock
Checking formatting...
app-config.local.yaml
app-config.yaml
catalog-info.yaml
yarn.lock[error] No parser could be inferred for file: yarn.lock
Code style issues found in the above file(s). Forgot to run Prettier?
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

And you'll notice it has all sorts of stuff like dist-types and other. But if I run it from the subdir from a cloned copy of Backstage, the list of items supplied is much smaller (and doesn't include dist-types, but WOULD include your oauth2 call back maybe)?


```
~/github/backstage/packages/app(master)*$ yarn prettier --check *
yarn run v1.22.1
$ /home/user/github/backstage/node_modules/.bin/prettier --check CHANGELOG.md README.md cypress cypress.json dist node_modules package.json public src
Checking formatting...
```

Sorry, don't intend this to be a [Yak shaving exercise](https://americanexpress.io/yak-shaving/) but thought I'd mention it.  😁

Reply:
I don't use create-app -- just keep pulling the main repo and have an app-config.local.yaml...

Reply:
So I don't think this is a fix to include for this PR, but it does seem like you'll want create-app to template .prettierignore